src/controllers/action-queue/util/delay.js

```diff
@@ -0,0 +1,3 @@
+// @flow
+
+export const delay = (ms: number): Promise<void> => new Promise(resolve => setTimeout(resolve, ms))
```

Comment:
Still duplicates `snooze` in `src/util/utils.js`.

Reply:
I am using `snooze`, I must of forgot to remove the file.